Signal/src/call/CallService.swift

```diff
@@ -253,34 +289,15 @@ private class SignalCallData: NSObject {
         }
     }
 
-    var call: SignalCall? {
+    // This accessor is intended for external use only.
+    // Internally, we should access the call through the call data.
+    public var call: SignalCall? {
         get {
             SwiftAssertIsOnMainThread(#function)
 
             return callData?.call
         }
     }
-    var peerConnectionClient: PeerConnectionClient? {
```

Comment:
I got rid of these convenience accessors in an effort to force us to be a little more strict about pulling data from `SignalCallData`.